package.json

```diff
@@ -83,9 +83,11 @@
     "eslint-plugin-prettier": "3.1.4",
     "fetch-mock": "9.11.0",
     "husky": "^8.0.3",
+    "jest": "^29.7.0",
```

Comment:
you don't need the library, bun is (closely) Jest compatible
https://bun.sh/docs/cli/test

Reply:
ok we do need this to get Jest support for the integration tests

Reply:
I'll remove it for now. We still don't have any integration tests..